ludwig/utils/data_utils.py

```diff
@@ -190,12 +198,22 @@ def read_xsv(data_fp, df_lib=PANDAS_DF, separator=",", header=0, nrows=None, ski
     if nrows is not None:
         kwargs["nrows"] = nrows
 
+    # If fetching a preview, create an iterator that will immediately be used to read the first chunk:
+    if "get_preview" in kwargs:
+        kwargs["chunksize"] = kwargs["get_preview"]
```

Comment:
[read_csv](https://docs.dask.org/en/stable/generated/dask.dataframe.read_csv.html) in Dask doesn't have `chunksize` but Pandas read_csv does.

Reply:
Per docs and disc.: Dask's version of the function will still use pandas and forward `chunksize` to it